src/components/ADempiere/FieldDefinition/FieldSearch/Invoice/PanelForm/QueryCriteria/OrderField.vue

```diff
@@ -18,14 +18,13 @@
 <template>
   <el-form-item :label="orderTitle">
     <el-select
-      v-model="saleOrderField"
-      clearable
+      v-model="currentSaleOrder"
       filterable
-      size="mini"
-      :filter-method="filterSearchOrder"
-      style="margin: 0px; width: 100%"
-      @visible-change="showList"
-      @change="currentValue()"
+      clearable
+      remote
+      :loading="isLoading"
+      :remote-method="remoteMethod"
+      @visible-change="showOrder"
     >
       <el-option
```

Comment:
please add `EmptyOption` component before `el-option` list.

Reply:
Manage as https://github.com/solop-develop/frontend-core/blob/feature/legacy-support/src/components/ADempiere/FieldDefinition/FieldSearch/ProductInfo/PanelForm/QueryCriteria/productCategoryField.vue#L38-L39 to improve peformance

Reply:
https://github.com/solop-develop/frontend-core/pull/2220/commits/64508a3abac781fa837483fd62e7f96c800c9b13